browser/src/App.js

```diff
@@ -93,7 +93,21 @@ const navLinks = [
   },
 ];
 
+const fetchContentCollection = () => {
+  const requestContentCollection = new Request('/content', {
+    method: 'GET',
+  });
```

Comment:
You don't need to do this at all.  Whenever you need to work with unfamiliar APIs, look up documentation -- https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch

Reply:
I was basing this off those docs, but perhaps putting the fetch request into a separate function like this is overkill/overly verbose?

Reply:
You just don't need to create an instance of the Request object at all.  I took a look at the docs and didn't see anything specifying that you had to do this -- do you mind pointing it out so I can try to explain why they were doing it?

Reply:
Ahh I see it here: https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch#supplying_your_own_request_object

Reply:
Using the Request constructor is there if you want to make a request for this data once and want to ensure it's only done once with that instance of the request.  In our case we don't need to worry about that at all, so you can just pass in the path to the resource you want to fetch.

Reply:
Ahh okay cool, thanks for clarifying!